README.md

```diff
@@ -182,13 +182,26 @@ limitd.takeElevated(type, key, { count, configOverride, erlIsActive }, (err, res
 -  `count`: the amount of tokens you need. This is optional and the default is 1.
 -  `configOverride`: caller-provided bucket configuration for this operation
 -  `erlIsActiveKey`: (string) the identifier of the ERL activation for the bucket.
+-  `erlQuota`: (object)
+  - `key`: (string) the identifier of the ERL quota bucket name.
```

Comment:
Key is a generic name that might cause confusion while looking at the code/configuring clients.

Reply:
Agree. I can change it to erlQuotaKey. WDYT? Also related to the previous comment

Reply:
fixed